packages/gallery/src/components/item/videos/videoItem.js

```diff
@@ -81,13 +97,60 @@ class VideoItem extends React.Component {
     return this.isHLSVideo() && utils.isiOS();
   }
 
+  updatePlayer = async () => {
+    if (!this.isYoutubeVideo()) {
+      return;
+    }
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
```

Comment:
This functionality should be injected using hooks or [HOC](https://github.com/wix-private/photography-pro-gallery/pull/909#discussion_r767546350). The work `youtube` should not be written here.